contrib/patterns/_grid-media.scss

```diff
@@ -0,0 +1,16 @@
+$medium-screen: 1000px;
+$medium: "only screen and (min-width:#{$medium-screen})";
```

Comment:
Oh, obvs but this is fakeish moc code and `$neat-grid:media` is meant to be the neat settings call for the `media`. but yea